packages/firebase_core/firebase_core/lib/firebase_core.dart

```diff
@@ -4,14 +4,7 @@
 
 library firebase_core;
```

Comment:
Now that this whole barrel file is defined with exports, do we need this library definition?

Reply:
No, good catch!